### PR TITLE
Feature/ukr 232 add get route for order

### PR DESCRIPTION
**Pull request by SecondeJK**
Additional route plus testing to get the Order.
Uses route model binding only rather that repository to keep it simple.

### CODE REVIEW

File: app/Providers/Service/InventoryOptionGroupItemServiceProvider.php [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+     *
+     * @return void
+     */
+    public function boot(): void
+    {
+        $this->app->bind(InventoryOptionGroupItemContract::class, function (Application $app) {
+            $logger = $app->make(LoggerInterface::class);
+
+            $inventoryOptionGroupItemRepository = $app->make(
+                \App\Contract\Repositories\InventoryOptionGroupItemContract::class
```
Comment (dannewnsjump24) [inline app/Providers/Service/InventoryOptionGroupItemServiceProvider.php:25]: Can we move this to be the shorthand notation here please for the namespace

File: app/Http/Controllers/Api/Order/ShowController.php [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+class ShowController extends Controller
+{
+    /**
+     * @param Order $order
+     * @param OrderContract $orderService
+     * @return JsonResponse
+     */
+    public function __invoke(Order $order): JsonResponse
+    {
+        return response()->json($order, Response::HTTP_OK);
```
Comment (dannewnsjump24) [inline app/Http/Controllers/Api/Order/ShowController.php:23]: Is this to show the whole order? how are we loading in items etc for this as there are no relationships loaded here?

File: app/Service/InventoryOptionGroupItemService.php [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+        LoggerInterface $logger
+    ) {
+        $this->inventoryOptionGroupItemRepository = $inventoryOptionGroupItemRepository;
+        $this->merchantRepository = $merchantRepository;
+        $this->logger = $logger;
+    }
+
+    public function validateOrderItemsBelongToMerchant(Collection $inventoryOptions, Merchant $merchant): bool
+    {
+        return $inventoryOptions->count() === $this->
```
Comment (dannewnsjump24) [inline app/Service/InventoryOptionGroupItemService.php:41]: You know i stared at this for a while thinking im not sure this is right but you know what, this feels like a really elegant solution for how to attack this problem. nice work @SecondeJK

File: app/Http/Controllers/Api/Order/ShowController.php (hunk 2 above)
Comment (SecondeJK) [reply app/Http/Controllers/Api/Order/ShowController.php:23]: At the moment this is -just- to return the order entity. I guess it's a design decision we need to make from a wider perspective when it comes to eager loading in relationships

**Pull request merged**